### PR TITLE
WT-10487 Using env for username and password for upload results

### PR DESCRIPTION
**Pull request by mickgraham-mongodb**

### CODE REVIEW

**Comment by mickgraham-mongodb** [comment]
We will have to time the merge with the fix in evergreen.yml.

File: largescale/upload-results-atlas.py [hunk 1]
```diff
@@ -53,4 +53,4 @@ def upload_results_atlas(coll, branch, filename):
 db = client[NS]
 coll = db[COLL]
 
-upload_results_atlas(coll, BRANCH, FILE_UPLOAD)
\ No newline at end of file
+upload_results_atlas(coll, BRANCH, FILE_UPLOAD)
```
Comment (etienneptl) [inline largescale/upload-results-atlas.py:56]: What is the change here?

**Review by etienneptl — APPROVED** [review]
LGTM, one question.

File: largescale/upload-results-atlas.py (hunk 1 above)
Comment (mickgraham-mongodb) [reply largescale/upload-results-atlas.py:64]: There wasn't one (no newline).

Comment (etienneptl) [reply largescale/upload-results-atlas.py:64]: Thanks, it did not show up on my phone.

**Review by etienneptl — APPROVED** [review]
LGTM!

**Pull request merged**